versioned_docs/version-1.2/administration/gitcatalog.mdx

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
`versioned_docs/{version}` is autogenerated by our framework when we create a new release. I think this need to go in `/src/content/administration`